### ISSUE TITLE
Calculator crashes on exit

### ISSUE THREAD

**Issue by nico** [SerenityOS/serenity#2702]
```
[Calculator(27:27)]: Process exec'd /bin/Calculator @ 0x08048ec0
WindowServer(13): Decoding a ShareableBitmap with shbuf_id=115, size=[16x16]
USERSPACE(27) ASSERTION FAILED: s_the
../Libraries/LibGUI/Application.cpp:46
[Terminal(24:24)]: Terminating due to signal 6
[FinalizerTask(3:3)]: 0xdeadc0de  Kernel::Processor::switch_context(Kernel::Thread*, Kernel::Thread*) +140
```

**Comment by awesomekling** [comment]
Can't repro. Looks like you're on an old build since there's no stack trace and that was fixed in bb84fad0bf3617ab78d027347e1c24ab1bf555a7 :)

**Comment by nico** [comment]
Can't repro either :shrug:
